packages/hub/src/app/models/[owner]/[slug]/utils.ts

```diff
@@ -43,22 +45,26 @@ export function entityNodes(
     },
   ];
 
-  if (variableName && variableNodeType === "RELATIVE_VALUE") {
+  if (variable && variable.type === "RELATIVE_VALUE") {
```

Comment:
This needs to be `variable?.name`, or you need to patch the code in `ModelLayout` and `RelativeLayout`.

Right now it shows an extra node with empty name and "undefined" in link on model pages:

https://quri-hub-git-modelexportspages-quantified-uncertainty.vercel.app/models/ozziegooen/errors
<img width="550" alt="Captura de pantalla 2023-11-13 a la(s) 17 30 31" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/ff09bf99-7b4d-4e2a-8b28-15a501db752c">

Reply:
gr... looks like useParams can return undefined, but is poorly typed. Investigating..